docs/xoa.md

```diff
@@ -61,7 +61,7 @@ Please only use this if you have issues with [the default way to deploy XOA](ins
 Alternatively, you can deploy it by connecting to your XenServer host and executing the following:
 
 ```
-bash -c "$(curl -sS https://xoa.io/deploy)"
+curl -sS https://xoa.io/deploy | bash
```

Comment:
Lots of unrelated changes, please remove.

Reply:
something went wrong with the merge

Reply:
It's not usually necessary to merge with `master`, don't worry if it says that the PR cannot be merged automatically, I can do it right before merging if it's trivial, otherwise I'll tell you.

Also, feel free to ask me to do the merge for you if you need help :slightly_smiling_face:

Reply:
I just removed the botched merge, thank you